### DISCUSSION TITLE
ggml-base.en.bin vs ggml-base.bin, what's the difference?

### DISCUSSION THREAD

**Discussion by chen-rn** [ggml-org/whisper.cpp#1460] [Q&A] [answered]
![Screenshot 2023-10-30 at 6 20 32 PM](https://github.com/ggerganov/whisper.cpp/assets/36214945/fa47a6d6-5883-4e8e-a5ac-884530966145)

Is the .en version faster/more accurate?

**Comment by chen-rn** [comment]
I realized this may be better as a discussion, feel free to move it there!

**Comment by bjnortier** [comment]
The .en models are for English only, and for the same size & speed they'll be more accurate for English

**Comment by bobqianic** [comment]
> they'll be more accurate for English

Do you have a source for that information? : )

AFAIK, smaller `.en` models tend to perform better for English, but this isn't necessarily the case for larger models. I'm not certain at what point this changes.

<img width="380" alt="image" src="https://github.com/ggerganov/whisper.cpp/assets/129547291/e9bf55bd-015b-4091-9a0e-e651424a5e65">

**Comment by bjnortier** [comment, marked as answer]
Interesting. If you look at the WER tables in the paper [1] (below):
- small.en performs better than small on 10 vs. 3 datasets for greedy
- small.en performs better than small on 11 vs. 2 datasets for beam.
- medium performs better than medium.en on 9 vs 4 datasets for greedy.
- medium performs better than medium.en on 9 vs. 5 datasets for beam.

![Screenshot 2023-11-09 at 10 44 52](https://github.com/ggerganov/whisper.cpp/assets/57994/3baed11f-263e-4b67-8a35-67cc4200acfe)
Greedy
|      | multi  | .en | same |
| ---- | ---- | --- | ---- |
|Small | 3    | 10  | 1    |
|Medium| 9    | 4   | 1    |


![Screenshot 2023-11-09 at 10 44 58](https://github.com/ggerganov/whisper.cpp/assets/57994/c8ae51f9-a8a8-4a75-aea5-97c54221922b)
Beam
|      | multi  | .en | same |
| ---- | ---- | --- | ---- |
|Small |  2   | 11  | 1    |
|Medium| 9    | 5   | 0    |

So it's probably better to use medium instead of medium.en for general cases. If you're transcribing telephone conversations (with beam) might be better to use medium.en because it performs better on CallHome and Switchboard. I guess that's why OpenAI released both medium.en and medium.

[1] https://cdn.openai.com/papers/whisper.pdf
